TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/PATCH-Negative-DeviceCommand.robot

```diff
@@ -13,30 +13,34 @@ ${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-deviceprofile
 *** Test Cases ***
 ErrProfileCommandPATCH001 - Update deviceCommand with device profile name validation error
     # non-existent device profile name
-    When Update deviceCommand with Non-existent device profile name
+    Given Generate a profile and command sample for updating
```

Comment:
Create profile is unnecessary for non-existent profile name.

Reply:
updated